ApplyToBecomeInternal/ApplyToBecomeInternal.Tests/Pages/ProjectNotes/AddProjectNoteIntegrationTests.cs

```diff
@@ -32,20 +35,24 @@ public async Task Should_navigate_to_add_note_from_project_notes_and_back_to_pro
 		public async Task Should_add_new_note_and_redirect_to_project_notes()
 		{
 			var project = AddGetProject();
-			AddGetProjectNotes(project.Id);
 
-			var projectNote = new AddProjectNote {Subject = _fixture.Create<string>(), Note = _fixture.Create<string>(), Author = "" };
-			AddPostProjectNote(project.Id, projectNote);
+         string projectNotesPage = $"/project-notes/{project.Id}";

```

Comment:
It looks like the indentation has gone awry here.

Reply:
I don't see what you mean - is it still visible in [full-file view](https://github.com/DFE-Digital/a2b-internal/blob/fab81413c79289b80582a9f6bac080d566d6a9a7/ApplyToBecomeInternal/ApplyToBecomeInternal.Tests/Pages/ProjectNotes/AddProjectNoteIntegrationTests.cs)?

Reply:
Sorry it was the view I had. My bad

Reply:
As much as I prefer GitHub to every other SCM service, it still has the odd foible :)